launcher/src/functionalTest/kotlin/com/cognifide/gradle/aem/launcher/LauncherTest.kt

```diff
@@ -9,7 +10,13 @@ import java.util.concurrent.TimeUnit
 class LauncherTest {
 
     @Test
-    fun shouldRunProperly() = makeTestProject("tasks", null) { launch("tasks") }
+    fun shouldRunProperly() = makeTestProject("tasks", null) {
+        launch("tasks")
+        assertEquals(
+            "https://services.gradle.org/distributions/gradle-7.5.1-bin.zip",
+            resolve("gradle/wrapper/gradle-wrapper.properties").readAsProperties()["distributionUrl"]
```

Comment:
The other tests are constructing `File`s relative to the cwd for this but that means hard-coding the project path. I had it like this as well but I forgot to update it after a copy-paste and thought this was cleaner. But I’m happy to revert it (or adjust the others if you agree).

Reply:
I am ok with any approach that is stable